packages/firebase_auth/firebase_auth_platform_interface/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_auth/firebase_auth_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.0
+version: 1.0.1
```

Comment:
This should be 1.1.0

Reply:
Ok, I accept all your suggestions and change version to 1.1.0.
Thanks